Yank/restraints.py

```diff
@@ -1738,6 +1738,10 @@ def determine_missing_parameters(self, thermodynamic_state, sampler_state, topog
             # Determine restraint parameters for these atoms.
             self._determine_restraint_parameters(sampler_state, topography)
 
+        # DEBUG
+        ssc = self.get_standard_state_correction(thermodynamic_state)
+        logger.debug('Standard state correction: {} kT'.format(ssc))
```

Comment:
Did you intend to get rid of this line before merging?

Reply:
No, I actually want that there, but I'll remove the `# DEBUG` comment.